docs/reference/search/rrf.asciidoc

```diff
@@ -1,8 +1,6 @@
 [[rrf]]
 === Reciprocal rank fusion
 
-preview::["This functionality is in technical preview and may be changed or removed in a future release. The syntax will likely change before GA. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features."]
```

Comment:
We just need to make sure that we CLEARLY communicate that `rank: rrf` is still tech preview and will be removed. I am not sure what things link to this doc page, but we need to be absolutely clear about that.

Reply:
All references to docs have been removed when we introduced the rrf retriever [here](https://github.com/elastic/elasticsearch/pull/105470#discussion_r1517535806). Should we add a warning header that `rank: rrf` is a now-deprecated option? 
I've also opened [this PR](https://github.com/elastic/elasticsearch/pull/114854) to add deprecation headers when using `rank: rrf` and/or `sub_searches`